source/class/qx/tool/cli/commands/MConfig.js

```diff
@@ -112,14 +116,6 @@ qx.Mixin.define("qx.tool.cli.commands.MConfig", {
       }
       this._mergeArguments(parsedArgs, config, lockfile_content);
 
-      if (config.libraries) {
-        for (const aPath of config.libraries) {
-          if (typeof aPath === "object" && typeof aPath.path === "string") {
```

Comment:
Can we simply remove this warning now - do you think no-one has the old format anymore? Or is it unnecessary since `qx migrate` has to be run anyways and take care of it?